server/views/referrals/new/oasysConfirmation/show.njk

```diff
@@ -55,7 +55,12 @@
           errorMessage: errors.messages.oasysConfirmed
         }) }}
 
-        {{ govukButton({ text: "Save and continue" }) }}
+        {{ govukButton({
+          attributes: {
+            'data-testid': 'oasys-confirmation-submit-button'
```

Comment:
Will use this for the e2e tests. Same goes for the change in the additional information template.